py/desispec/io/filters.py

```diff
@@ -23,3 +24,34 @@ def read_filter_response(given_filter,basepath):
     ## filter_response=(wave, throughput, tck)
     ## return filter_response
     return wave, throughput
+
+def load_filter_response(given_filter):
+    """ 
```

Comment:
Does it make sense to use the names already defined by `speclite.filters` so we can take advantage of their docs directly (http://speclite.readthedocs.org/en/latest/filters.html) instead of documenting these translation rules.